.github/workflows/test.yml

```diff
@@ -38,8 +38,8 @@ jobs:
           pip install -e .[dev,cfg,z3]
       - name: Run tests
         run: |
-          pytest --cov python_ta --cov-config=.coveragerc --cov-report lcov --ignore=tests/test_type_constraints --ignore=tests/test_type_inference tests --ignore=tests/test_debug
-          pytest tests/test_debug
+          pytest tests --cov python_ta --cov-config=.coveragerc --cov-report lcov --ignore=tests/test_type_constraints --ignore=tests/test_type_inference --ignore=tests/test_debug
+          pytest tests/test_debug -vv --ignore=tests/test_debug/test_accumulation_table.py
```

Comment:
You don't need an `ignore` here, and instead you _should_ modify the `tests/test_debug` part so that the _only_ tests that are run are the ones in `test_accumulation_table.py`